### PR TITLE
Latest MHR API schema updates.

### PR DESCRIPTION
**Pull request by doug-lovett**
Signed-off-by: Doug Lovett <doug@diamante.ca>

*Issue #:* /bcgov/entity#14864

*Description of changes:*
- MHR owner schema add partyType, description to support transfer due to death.
- MHR searchSummary schema add activeCount, exemptCount, historicalCount.
- PPR searchSummary schema add missing registration types "MD", "PT", "SC", "TO", "SV".

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the registry-schemas license (Apache 2.0).

### CODE REVIEW

File: src/registry_schemas/schemas/ppr/searchSummary.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
                     "type": "string",
                     "maxLength": 2,
                     "enum": ["SA", "RL", "FR", "LT", "MH", "SG", "FL", "FA", "FS", "MR", "CC", "CT", "DP", "ET", "FO", "FT", "HR", "IP", "LO", "MI", "OT"
-                        , "PG", "PS", "IT", "RA", "SS", "TL", "HN", "ML", "MN", "PN", "WL", "TF", "TA", "TG", "TM"],
+                        , "PG", "PS", "IT", "RA", "SS", "TL", "HN", "ML", "MN", "PN", "WL", "TF", "TA", "TG", "TM", "MD", "PT", "SC", "TO", "SV"],
```
Comment (cameron-eyds) [inline src/registry_schemas/schemas/ppr/searchSummary.json:32]: What do these Financial Statement Codes correlate too?

**Review by cameron-eyds — APPROVED** [review]
Looks great!

**Review by dimak1 — APPROVED** [review]

File: src/registry_schemas/schemas/ppr/searchSummary.json (hunk 1 above)
Comment (doug-lovett) [reply src/registry_schemas/schemas/ppr/searchSummary.json:32]: They are the set of PPR registration types. The base registration is a Financing Statement.

Comment (cameron-eyds) [reply src/registry_schemas/schemas/ppr/searchSummary.json:32]: copy that, thank you!

**Pull request merged**